### PR TITLE
[hi] localize contributor-ladder/_index.md

### PR DESCRIPTION
**Pull request by Garima-Negi**
- This PR localizes the contribution ladder page.
- Related to issue [https://github.com/cncf/glossary/issues/270](https://github.com/cncf/glossary/issues/270)

### CODE REVIEW

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:10]:
```suggestion
नमस्ते! 👋 CNCF क्लाउड नेटिव शब्दावली परियोजना में योगदान करने में आपकी रुचि के लिए धन्यवाद। इस समुदाय के सक्रिय सदस्य बनने के कई तरीके हैं जैसे नए शब्दों का योगदान देना, शब्दावली को अपनी मूल भाषा में स्थानीयकृत करने में मदद करना, या दूसरों को योगदान आरंभ करने में मदद करना। यह दस्तावेज़ परियोजना के भीतर विभिन्न योगदानकर्ता भूमिकाओं और उनके साथ आने वाली जिम्मेदारियों और विशेषाधिकारों की रूपरेखा की व्याख्यान करता है।
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:14]:
```suggestion
यह शब्दावली सभी के लिए है। कोई भी व्यक्ति केवल परियोजना में योगदान करके शब्दावली योगदानकर्ता बन सकता है। सभी योगदानकर्ताओं से [CNCF आचार संहिता](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) का पालन करने की अपेक्षा की जाती है।
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:21]:
```suggestion
- **विषय योगदानकर्ता**: हर कोई जो मौजूदा शब्दों में सुधार करता है या नए योगदान देता है, 
- **स्थानीयकरण योगदानकर्ता**: जो शब्दकोष का अन्य भाषा में अनुवाद करने में मदद करते हैं,
- **सहायक**: कोई भी व्यक्ति जो GitHub, Slack पर या जहाँ भी समुदाय के सदस्यों को समर्थन की आवश्यकता है, दूसरों की मदद करता है,
- **दूत (Ambassadors)**: जो कोई भी इस बात को फैलाने में मदद करता है, समुदाय को शिक्षित करता है कि कैसे और क्यों योगदान करना चाहिए। 
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:24]:
```suggestion
योगदानकर्ताओं की कई भूमिकाएँ हो सकती हैं या वे केवल एक क्षेत्र पर ध्यान केंद्रित कर सकते हैं। **ये सभी योगदान समान रूप से महत्वपूर्ण हैं** और एक संपन्न समुदाय को बढ़ावा देने में मदद करते हैं। कृपया विषय और स्थानीयकरण योगदान के लिए [योगदान कैसे करें](https://glossary.cncf.io/contribute/) और [शैली मार्गदर्शिका](https://glossary.cncf.io/style-guide/) देखें।
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:30]: We should decide on one, अप्रूवर or अनुमोदनकर्ता or स्वीकृतकर्ता. Currently, we have all three being used on this page.
 
My suggestion would be to switch to अप्रूवर everywhere except for first line `अप्रूवर या अनुमोदनकर्ता, PR (पुल अनुरोध) पर प्रतिक्रिया प्रदान` as अनुमोदनकर्ता is not a very rarely used term.

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:47]:
```suggestion
**तकनीकी  अप्रूवर**: एक मजबूत तकनीकी पृष्ठभूमि वाले व्यक्ति, ठोस अंग्रेजी लेखन कौशल के बिना भी अनुमोदनकर्ता हो सकते हैं। हालांकि, अगर वे तकनीकी योग्यता के आधार पर PR को मंजूरी देते हैं, तो उन्हें यह सुनिश्चित करना होगा कि इसकी समीक्षा एक (संपादक) अप्रूवर द्वारा की जाए।
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:49]:
```suggestion
**संपादक**: संपादक शब्दों का प्रूफरीड करते हैं और सुनिश्चित करते हैं कि उन्हें शैली मार्गदर्शिका के अनुसार सरल भाषा में समझाया गया है। यदि कोई शब्द भारी रूप से संपादित किया गया है, तो संपादक को तकनीकी अप्रूवर से अनुरोध करना चाहिए कि वह इसकी फिर से समीक्षा करें ताकि यह सुनिश्चित हो सके कि अर्थ में कोई बदलाव नहीं किया गया है।
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:55]:
```suggestion
### अप्रूवर बनना 
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:57]:
```suggestion
स्वीकृति देने वाले उम्मीदवारों के पास उच्च गुणवत्ता वाले पुल अनुरोध जमा करने का एक सिद्ध अभिलेख होना चाहिए और दूसरों को अपने पुल अनुरोध को विलय योग्य स्थिति में लाने में मदद करना चाहिए। यदि उनका समय क्षेत्र अनुमति देता है, तो उन्हें नियमित रूप से [शब्दावली कार्य समूह की बैठकों](https://www.cncf.io/calendar/) में भी भाग लेना चाहिए।
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:59]:
```suggestion
अप्रूवर बनने के लिए, मौजूदा मेन्टेनेरों को अपनी रुचि व्यक्त करके प्रारंभ करें। इसके बाद मौजूदा मेन्टेनेर आपसे पुल अनुरोध में योगदान करके, समीक्षा करके, और उनके मार्गदर्शन में ऐसे अन्य कार्यों को करके उपरोक्त योग्यताओं को प्रदर्शित करने के लिए कहेंगे। एक साथ काम करने के कुछ समय बाद, मेन्टेनेर्स यह तय करेंगे कि आपको अप्रूवर का दर्जा देना है या नहीं। यह निर्णय आपकी दक्षता और जवाबदेही के प्रदर्शित स्तर पर आधारित होगा।
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:68]:
```suggestion
- एक सक्रिय और उत्तरदायी अप्रूवर बनें (ऊपर देखें),
- साइट कॉन्फ़िगरेशन, अनुमति, इश्यू-टेम्पलेट, गिटहब वर्कफ़्लो सहित अन्य लोगों के बीच रिपोजिटरी को मेन्टेन करने में सहायता करें,
- शब्दावली Slack चैनलों की निगरानी करें और जब भी संभव हो मदद करें,
- नियमित रूप से [शब्दावली कार्य समूह की बैठकों](https://www.cncf.io/calendar/) में भाग लें (यदि समय क्षेत्र अनुमति देता है)
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:76]:
```suggestion
एक मेन्टेनेर बनने के लिए, मौजूदा मेन्टेनेरों को अपनी रुचि व्यक्त करके प्रारंभ करें। इसके बाद मौजूदा मेन्टेनेर्स आपसे पुल अनुरोध में योगदान करके, समीक्षा करके, और उनके मार्गदर्शन में ऐसे अन्य कार्यों को करके उपरोक्त योग्यताओं को प्रदर्शित करने के लिए कहेंगे। एक साथ काम करने के कुछ समय बाद, मेन्टेनेर्स यह तय करेंगे कि मेन्टेनेर का दर्जा देना है या नहीं। यह निर्णय दक्षता और जवाबदेही के प्रदर्शित स्तर पर आधारित होगा। 
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:80]:
```suggestion
सामुदाय प्रबंधक एक स्वागत योग्य और आकर्षक समुदाय को बढ़ावा देने में मदद करते हैं। समुदाय का कोई भी सदस्य समुदाय प्रबंधक बन सकता है। उनसे अपेक्षा की जाती है:
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:84]:
```suggestion
- नए सदस्यों का स्वागत करें और सुनिश्चित करें कि उन्हें वह जानकारी मिले जिसकी उन्हें आवश्यकता है,
- समुदाय के सवालों के जवाब देने में मदद करें या किसी ऐसे व्यक्ति की पहचान करें जो मदद कर सकता है,
- Slack पर बातचीत मध्यस्थ करें।
```

File: content/hi/contributor-ladder/_index.md
Comment (anubha-v-ardhan) [inline content/hi/contributor-ladder/_index.md:88]:
```suggestion
कोई भी शब्दकोष समुदाय प्रबंधक बन सकता है। सामुदाय प्रबंधकों को योगदान और स्थानीयकरण प्रक्रिया की एक ठोस समझ होनी चाहिए और बातचीत करने और दूसरों की मदद करने का आनंद लेना चाहिए। सामुदाय प्रबंधक बनने के लिए, मौजूदा मेन्टेनेरों को अपनी रुचि व्यक्त करके प्रारंभ करें। ऑनबोर्डिंग/परीक्षण अवधि के बाद, मेन्टेनेर्स यह निर्णय लेंगे कि प्रदर्शन के आधार पर समुदाय प्रबंधक का दर्जा देना है या नहीं। 
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thankyou @Garima-Negi :)
Please go through my suggestions here and commit them if you agree. Feel free to discuss if you feel like a change is unnecessary or inaccurate.

File: content/hi/contributor-ladder/_index.md
Comment (jayesh-srivastava) [reply content/hi/contributor-ladder/_index.md:30]: +1 to अप्रूवर

File: content/hi/contributor-ladder/_index.md
Comment (jayesh-srivastava) [inline content/hi/contributor-ladder/_index.md:63]: I think the spelling of maintainers will be मेन्टेनर्स

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

File: content/hi/contributor-ladder/_index.md
Comment (Garima-Negi) [reply content/hi/contributor-ladder/_index.md:21]: agreed.

File: content/hi/contributor-ladder/_index.md
Comment (Garima-Negi) [reply content/hi/contributor-ladder/_index.md:76]: good suggestion.

File: content/hi/contributor-ladder/_index.md
Comment (Garima-Negi) [reply content/hi/contributor-ladder/_index.md:63]: We've established maintainer as मेन्टेनर so मेन्टेनेरों should be easily understandable for the reader

File: content/hi/contributor-ladder/_index.md
Comment (Garima-Negi) [reply content/hi/contributor-ladder/_index.md:83]: thank you for this correction!

File: content/hi/contributor-ladder/_index.md
Comment (Garima-Negi) [reply content/hi/contributor-ladder/_index.md]: Done.

File: content/hi/contributor-ladder/_index.md
Comment (Garima-Negi) [reply content/hi/contributor-ladder/_index.md]: done

File: content/hi/contributor-ladder/_index.md
Comment (jayesh-srivastava) [inline content/hi/contributor-ladder/_index.md:60]: We can add maintainers in parenthesis just like the approver has it.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]
Just one little change, rest all lgtm!

File: content/hi/contributor-ladder/_index.md
Comment (Garima-Negi) [reply content/hi/contributor-ladder/_index.md]: done

**Review by jayesh-srivastava — APPROVED** [review]
LGTM!! Thanks @Garima-Negi

**Pull request merged**
